exercise-1.js

```diff
@@ -380,60 +507,98 @@
                 "tech_stack" : null
             },
         ];
+    
+        let bio = devs.forEach(console.log((a), a.name + " specializes in " + a["tech_stack"]))
```

Comment:
Error => let bio = devs.forEach(console.log(devs.name + " specializes in " + devs["tech_stack"]))
                       ^

TypeError: undefined is not a function
    at Array.forEach (<anonymous>)
    at Object.<anonymous> (c:\Users\mbarbosa\Desktop\Team Marcelo\nichols.zane\JavaScriptZN\tempCodeRunnerFile.js:64:24)
    at Module._compile (node:internal/modules/cjs/loader:1108:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1137:10)
    at Module.load (node:internal/modules/cjs/loader:973:32)
    at Function.Module._load (node:internal/modules/cjs/loader:813:14)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:76:12)
    at node:internal/main/run_main_module:17:47